### PR TITLE
Iow 534 wdfn graph server update puppeteer

### PR DESCRIPTION
**Pull request by abriggs-usgs**
Before making a pull request
----------------------------

-   [x] Make sure all tests run
-   [x] Run the npm run lint command locally and verify that your code passes.
-   [x] Update the changelog appropriately

Description
-----------
This updates Puppeteer to the most recent version. 

So . . . after much  go around, the issue that was preventing Puppeteer from working was that the internal browser was not available to the application. Digging into Node_modules folder showed that the file was in the correct location, but it was failing to be unzipped.  Since the file was not available to the application in the expected format, the application attempted to reach the file over and over again causing to it to fall into an endless loop until the process was stopped with a timeout error from the browser. 

Once the issue was known, I was able to do an internet search for the solution and found https://github.com/puppeteer/puppeteer/blob/main/docs/troubleshooting.md#chrome-is-downloaded-but-fails-to-launch-on-nodejs-14 . This indicated that there was an issue with version 14xx of node and the solution was to use version 14.1.0. 

The application was using version 12.16.3 with the highest long term support version of node currently at 12.19.0. So it is surprising that the solution was to use a 'beta' version. However, that was indeed the solution. 

Using node 14.1.0 allowed for the successful updating of all dependencies including Puppeteer. 

After the dependencies were updated, I tested the Docker container version of the application, and it now runs successfully as well. 

After making a pull request
---------------------------

-   [ ] If appropriate, put the link to the PR in the JIRA ticket
-   [ ] Assign someone to review unless the change is trivial

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -24,11 +24,34 @@ last seven days. The query parameter parameterCode is required and should be a v
...
+Lower versions of Node will not unzip the internal Chromium package 
+correctly and this will cause the graph server to 'hang' while it attempts to access a file that it will never find.
+
+
 The dependencies need to be installed. To clear out any existing dependencies and to install:
 ```bash
 % npm run clean
 % npm install
 ```
+The default port will be 2929 and the base of the URL will start with api/graph-images.
```
Comment (mbucknell) [inline README.md:37]: Nice additions!

File: Dockerfile [hunk 2]
```diff
@@ -1,5 +1,5 @@
 # Use LTS release
-FROM node:12-slim
+FROM node:14.1.0
```
Comment (mbucknell) [inline Dockerfile:2]: I mentioned in chat, but it would be best to not pin the version here and if the -slim image works we should continue to use that.

**Review by mbucknell — APPROVED** [review]
Great job tracking this down!

File: Dockerfile (hunk 2 above)
Comment (abriggs-usgs) [reply Dockerfile]: Will do. ✔

**Pull request merged**